rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -3392,6 +3454,19 @@ bool ppu_initialize(const ppu_module& info, bool check_only)
 				}
 			}
 
+			if (jit)
+			{
+				const auto far_jump = ppu_get_far_jump(func.addr) ? g_fxo->get<ppu_far_jumps_t>().gen_jump(func.addr) : nullptr;
+
+				if (far_jump)
+				{
+					// Replace the function with ppu_far_jump
```

Comment:
I don't think it works as intended. Module-local jumps don't use the global mapping.

Reply:
Yes but it updates its PPU hash and forces recompilation. That's how it was handled before.

Reply:
That's good but how forcing recompilation would handle local branches? Especially with PPUTranslator having far jump part removed.

Reply:
It uses the global table now if I'm not mistaken because the function is removed from llvm, that's why it installs the LLVM value manually.

Reply:
The code below is responsie for its compilation in llvm and that's skipped.

Reply:
Okay, I see now.